### ISSUE TITLE
Screenreaders are not announcing the Helper Text

### ISSUE THREAD

**Issue by Paul-Grogan** [carbon-design-system/carbon#6988]
## Environment

> Operating system

Windows 10

> Browser

Firefox (81.0.1 and ESR78.3.1)

> Assistive technology used to verify

JAWS2020 and VoiceOver (on an iOS 13 device)

## Detailed description

The helper text on an input (e.g. the Text Input) is not getting announced by the screenreaders. 
This is resulting in potentially important information being unavailable to screenreader users.
See the attached video for a demonstration on the Text Input example ( https://www.carbondesignsystem.com/components/text-input/usage#live-demo )

> What version of the Carbon Design System are you using?

According to the footer on the Text Input example, Vanilla Components version ^10.21.0-rc.1

> What did you expect to happen?

Expected the Label, Placeholder AND Helper Text to get announced when the text input field became focused.

> What happened instead?

The Label and Placeholder got announced. The Helper Text did not.

> What WCAG 2.1 checkpoint does the issue violate?

Guideline 3.3 Input Assistance ( https://www.w3.org/TR/WCAG21/#input-assistance )

## Steps to reproduce the issue

1. Launch the screen reader
2. Launch the browser
3. Navigate the browser to https://www.carbondesignsystem.com/components/text-input/usage#live-demo
4. Using tab, place focus in the text input to get the announcement.

## Additional information

- Video
[TextInputHelperTextNotAnnounced.zip](https://github.com/carbon-design-system/carbon/files/5341850/TextInputHelperTextNotAnnounced.zip)
